code-challenges.test.js

```diff
@@ -22,10 +22,48 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisible by three"
 
-
+// create a test that fails
+// it decides if a number is divisible by 3 or not
+// if the variables are divisible
+describe ('divisible3', ()=>{
+  test('decides if a number is evenly divisible by three or not', ()=>{
+    expect(divisible3(num1)).toEqual("15 is divisible by three")
+    expect(divisible3(num2)).toEqual("0 is divisible by three")
+    expect(divisible3(num3)).toEqual("-7 is not divisible by three")
+  })
+})
+
+//Got RED
 
 // b) Create the function that makes the test pass.
 
+//create a function that takes in a variable
+//if the number is divisible by three return expected output based on prompt
+//else return false Boolean value
+
+const divisible2 = (number)=>{
+  if (number % 3 === 0){
+    return "15 is divisible by 3"
+  }else {
+    return "false"
+  }
+}
+const divisible1 = (number)=>{
+  if (number % 3 === 0){
+    return "0 is divisible by 3"
+  }else {
+    return "false"
+  }
+}
+const divisible3 = (number)=>{
+  if (number % 3 === 0){
+    return "-7 is not divvisible by 3"
+  }else {
+    return "false"
+  }
+}
```

Comment:
This is really good work here and you have all the basics of whats going to be needed down the road! You've probably figured out by now that we only want to write a function once and have it handle our different inputs instead of writing it for each input we have.

Reply:
```javascript
const divisible2 = (number)=>{
  if (number % 3 === 0){
    return number + " is divisible by 3"
  }else {
    return number + "is not divisible by three"
  }
}
```